helm/sloop/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+  {{- range $key, $value := .Values.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.name }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+  name: {{ .Values.name }}
+  #namespace: {{ .Values.namespace }}
+spec:
+  rules:
+  - host: {{ .Values.ingress.host }}
+    http:
+      paths:
+      - backend:
+          serviceName: {{ .Values.name }}
+          servicePort: {{ .Values.service.port }}
+        path: /
+{{- end -}}
+  {{- if .Values.ingress.tls }}

```

Comment:
just curious, ingress.tls is a {}, how does this work in if condition?

Reply:
if you remove {} it should accept kubernetes ingress format, like this:

```
 tls:
  - secretName: sloop-tls
    hosts:
     - sloop.example.com
```

Also a examplo from kubernetes-dashboard:
https://github.com/helm/charts/blob/7e45e678e39b88590fe877f159516f85f3fd3f38/stable/kubernetes-dashboard/values.yaml#L149